.github/workflows/deploy_infra.yml

```
@@ -42,41 +42,25 @@ jobs:
       - name: Checkout
         uses: actions/checkout@8e5e7e5ab8b370d6c329ec480221332ada57f0ab
 
-      - name: Setup node
-        uses: actions/setup-node@64ed1c7eab4cce3362f8c340dee64e5eaeef8f7c
-        with:
-          node-version-file: '.node-version'
-          cache: 'npm'
+      - name: Setup Node.JS
+        uses: ./.github/actions/setup-node
 
       - name: Install dependencies
-        run: npm ci
+        run: npm ci --audit=false --fund=false
 
-      - name: Compile
-        run: npm run compile
+      - name: Compile Cloudfront Functions
+        run: npm run compile -w cloudfront-functions
 
       - name: Configure AWS Credentials
-        env:
-          AWS_REGION: eu-south-1
-        uses: aws-actions/configure-aws-credentials@67fbcbb121271f7775d2e7715933280b06314838
+        uses: ./.github/actions/configure-aws-credentials
         with:
-          role-to-assume: ${{ secrets.IAM_ROLE }}
-          aws-region: ${{ env.AWS_REGION }}
-
-      - name: Set terraform version
-        run: |
-          echo "TERRAFORM_VERSION=`cat .terraform-version`" >> $GITHUB_ENV
+          aws_region: 'eu-south-1'
```

Same here; take the `aws_region` from env if defined. Otherwise it uses `eu-south-1` as default.

I'm not sure, but I think it can be done as follows:

```suggestion
          aws_region: ${{ env.AWS_REGION || 'eu-south-1' }}
```